### PR TITLE
feat: make OPL guide Console first

### PR DESCRIPTION
**Pull request by hperl**
This PR adds instructions on how to use the OPL on the Ory Network and shifts the instructions on how to do this locally further down the article.

### CODE REVIEW

File: docs/keto/guides/userset-rewrites.mdx [hunk 1]
```diff
@@ -1,56 +1,54 @@
...
-After you created both configuration files (`keto.yaml` and `namespaces.keto.ts`), run this command to start Ory Keto:
+Next, you need to connect to the Ory Network using the Keto CLI. First, [install Keto](../install.mdx). To authenticate, navigate
+to `Access & APIs` in the Ory Console and create a Personal Access Token. This, together with the subdomain from the SDK
+configuration URL, need do be exported in a shell:
 
 ```sh
-$ keto serve --config ./path/to/keto.yaml
+export KETO_READ_REMOTE=<your project slug>.projects.oryapis.com:443
+export KETO_WRITE_REMOTE=<your project slug>.projects.oryapis.com:443
+export KETO_BEARER_TOKEN="<your access token>"
```
Comment (aeneasr) [inline docs/keto/guides/userset-rewrites.mdx:46]: Could we instead use the Ory CLI for this?

**Comment by tomekpapiernik** [comment]
Things that aren't okay IMO: 
- Checking permissions is a big part of the document but in the current state, it (running checks) all has to be done using the Keto CLI. Seems to me that the Ory CLI doesn't allow that just yet. Is it worth updating the document to re-focus it for Ory Network if the user still has to rely on the Keto CLI? 
- `Connecting to the Ory Permission Language via CLI` - this doesn't really make sense, does it? What I see happening is the Keto CLI getting "connected" to an Ory Network project. The permission language is a way to define relations, not something you can connect to, right? 
- If you want to include screenshots, make sure they are of good quality and focus on the UI elements that are of interest in the given context. In your case, the left side of the screenshot has a black vertical line and too much blank space included in the view. You can make the screenshot more focused by resizing the window - this will cut down on the empty spaces and help focus on what's important.

File: docs/keto/guides/userset-rewrites.mdx (hunk 1 above)
Comment (hperl) [reply docs/keto/guides/userset-rewrites.mdx:46]: The Ory CLI currently does not support check yet. I'm all for revisiting the guide once that is implemented.

File: docs/keto/guides/userset-rewrites.mdx [hunk 2]
```diff
@@ -1,56 +1,59 @@
...
 
 import CodeFromRemote from "@theme/CodeFromRemote"
+import BrowserWindow from "@site/src/theme/BrowserWindow"
 const file = (path) => `https://github.com/ory/keto/blob/master/contrib/rewrites-example/${path}`
 
-This guide explains how to configure namespaces and relations using the
-[Ory Permission Language](../reference/ory-permission-language).
+This guide explains how to configure namespaces and relationship rules using the
+[Ory Permission Language](../reference/ory-permission-language) (OPL). You can then run fine-grained checks against Ory Keto,
+which are answered based on a combination of OPL and the concrete high-level relationships stored.
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:13]: > You can then run fine-grained checks against Ory Keto
If you start from scratch, you won't be doing anything against Keto until you complete the optional step which is running a Keto instance, correct? The permission checks are handled by Ory Permissions in Ory Network if I'm not mistaken.

File: docs/keto/guides/userset-rewrites.mdx [hunk 3]
```diff
@@ -1,56 +1,59 @@
...
 
 The example describes a file store. Individual files are organized in a folder hierarchy, and can be accessed by individual users
 or groups of users. Using the Ory Permission Language you can specify that if a user has access to a folder, the user also has
 access to all files in that folder.
 
-## Setup and Configuration
+## Ory Network Setup
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:19]:
```suggestion
## Ory Network setup
```

We moved away from title case in headings and titles a while back ;)

File: docs/keto/guides/userset-rewrites.mdx [hunk 4]
```diff
@@ -1,56 +1,59 @@
...
 
-First, [install Keto](../install.mdx). Next, create a `keto.yaml` file with this basic content:
+The fastest way to get started with Ory Permissions is using [Ory Console](https://console.ory.sh/). You can register for an Ory
+Network account for free.
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:22]: Now this sentence here is a bit weird - the title suggests this is all about OPL and then you're saying that the easiest way to start with Ory Permissions is xxx.

File: docs/keto/guides/userset-rewrites.mdx [hunk 5]
```diff
@@ -1,56 +1,59 @@
...
 
-<CodeFromRemote src={file("keto.yaml")} title="keto.yaml" />
+After logging in, navigate to `Namespaces & Rules` in the sidebar, and to the `Permisson Rules` tab.
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:24]:
```suggestion
In the Ory Console, select **Namespaces & Rules** from the navigation bar and switch to the **Permission Rules** tab.
```

Comment (tomekpapiernik) [reply docs/keto/guides/userset-rewrites.mdx:24]: The UI path description is now more in line with what I'm trying to do, UI element names are in bold (again, something I'm trying to introduce as a standard), typo fixed.

File: docs/keto/guides/userset-rewrites.mdx [hunk 6]
```diff
@@ -1,56 +1,59 @@
...
-Namespace configurations without the Ory Permission Language are still supported by either specifying the namespaces directly or a
-"naked" URI, e.g. `namespaces: file://.namespaces.yaml`.
+</BrowserWindow>
 
-:::
-
-Next, create a file with the namespace configuration:
+Into the editor, paste the following content:
 
 <CodeFromRemote src={file("namespaces.keto.ts")} title="namespaces.keto.ts" />
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:34]: This should be wrapped in mdx-code-block I think

File: docs/keto/guides/userset-rewrites.mdx [hunk 7]
```diff
@@ -1,56 +1,59 @@
...
-In the `namespaces` key, you see the line `config: file://./namespaces.keto.ts`. This needs to point to your namespace
-configuration in the Ory Permission Language.
+<BrowserWindow url="https://console.ory.sh/projects/3afd6339-b999-4649-a2f6-740736775f80/permissions/configuration">
 
-:::info
+![Custom Domain Cors disabled](../_static/opl_editor.png)
 
-Namespace configurations without the Ory Permission Language are still supported by either specifying the namespaces directly or a
-"naked" URI, e.g. `namespaces: file://.namespaces.yaml`.
+</BrowserWindow>
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:30]: This should be wrapped in mdx-code-block I think

File: docs/keto/guides/userset-rewrites.mdx [hunk 8]
```diff
@@ -114,6 +117,42 @@ $ keto check User:Henning edit File private
 Allowed
 ```
 
+## (Optional) Local Setup and Configuration
+
+If you do not want to use the Ory Console you need to [install Keto](../install.mdx) locally. Next, create a `keto.yaml` file with
```
Comment (tomekpapiernik) [inline docs/keto/guides/userset-rewrites.mdx:122]: > If you do not want to use the Ory Console

I think it's also about not using Ory Permissions but instead relying on a self-hosted instance, right?

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]
Love the new screenshot presentation method! <3
Still some more tweaking to do though ;)

File: docs/keto/guides/userset-rewrites.mdx (hunk 2 above)
Comment (hperl) [reply docs/keto/guides/userset-rewrites.mdx:13]: Well you don't have to do anything if you use the Ory Network. If you want to self-host, there's a section below.

File: docs/keto/guides/userset-rewrites.mdx (hunk 4 above)
Comment (hperl) [reply docs/keto/guides/userset-rewrites.mdx:22]: How's that weird? The OPL is a subset of Ory Permissions, and to use the OPL you need to use Ory Permissions, and the easiest way to start with that that is the Ory Network.

What would you like to read here?

File: docs/keto/guides/userset-rewrites.mdx (hunk 7 above)
Comment (hperl) [reply docs/keto/guides/userset-rewrites.mdx:30]: Why? This is not code we will show to the user, but code to generate the browser window. I'm confused.

File: docs/keto/guides/userset-rewrites.mdx (hunk 8 above)
Comment (hperl) [reply docs/keto/guides/userset-rewrites.mdx:122]: Yes.

**Review by tomekpapiernik — APPROVED** [review]

**Pull request merged**